awx/main/management/commands/custom_venvs.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2015 Ansible, Inc.
+# All Rights Reserved
+
+import json
+
+from awx.main.utils.common import get_custom_venv_choices, get_custom_venv_pip_freeze
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Returns either a list of custom venv paths or outputs the pip freeze from the path passed in the argument"""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--path',
+            dest='path',
+            type=str,
+            default='',
+            help='run without arguments to see a list of paths, run with one of those paths as an argument and see the pip freeze data',
+        )
+
+    def handle(self, *args, **options):
+        super(Command, self).__init__()
+        if options.get('path'):
+            pip_data = get_custom_venv_pip_freeze(options.get('path'))
+            print(pip_data)
+        else:
+            venvs = get_custom_venv_choices()
+            for venv in venvs:
+                print(venv)
```

Comment:
how does the user know what line corresponds to what custom venv?

Reply:
only one venv path can be passed at a time

Reply:
ah, sorry I misunderstood, the path is the only info shared here, I don't know that there's a good way to demonstrate any more than that which path corresponds to which venv identity

Reply:
oh right, yeah that flew past me.

So this usage doesn't show any pip freeze data.

Reply:
if you pass no path in the command it spits out a list of paths for you to view, if you pass one of those paths in it will spit out the pip freeze data